command.go

```diff
@@ -2125,6 +2126,24 @@ func commandInfoParser(rd *proto.Reader, n int64) (interface{}, error) {
 	}
 	cmd.StepCount = int8(stepCount)
 
+	_, err = rd.ReadReply(func(rd *proto.Reader, n int64) (interface{}, error) {
+		cmd.ACLFlags = make([]string, n)
+		for i := 0; i < len(cmd.Flags); i++ {
```

Comment:
Thank you for your support.
I found a mistaken in L2131. cmd.Flags => cmd.ACLFlags.